dali/test/python/test_dali_cpu_only.py

```diff
@@ -764,7 +926,9 @@ def test_arithm_ops_cpu():
 def test_arithm_ops_cpu_gpu():
     pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=None)
     data = fn.external_source(source = get_data, layout = "HWC")
-    processed = [data * data.gpu(), data + data.gpu(), data - data.gpu(), data / data.gpu(), data // data.gpu()]
+    processed = [data * data.gpu(), data + data.gpu(), data - data.gpu(), data / data.gpu(), data // data.gpu(), data ** data.gpu(),
```

Comment:
nitpick: consider writing the list in a column for better readability.

Reply:
Done